### PR TITLE
Add com.github.ilai_deutel.kibi

### PR DESCRIPTION
**Pull request by ilai-deutel**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - Kibi is a text editor, therefore read-write filesystem access is required
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - GitHub
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.ilai_deutel.kibi.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: kibi
+finish-args:
+  - --filesystem=host
+  - --filesystem=/tmp
+  - --filesystem=/var/tmp
```
Comment (hfiguiere) [inline com.github.ilai_deutel.kibi.yml:11]: it doesn't display any GUI?

**Comment by barthalion** [comment]
bot, build com.github.ilai_deutel.kibi

**Comment by barthalion** [comment]
Note that we currently don't display CLI applications on the store at all. If you still would like to distribute Kibi on Flathub, you can add a `flathub.json` file with `skip-appstream-check` key like this one: https://github.com/flathub/com.valvesoftware.Steam.CompatibilityTool.Proton/commit/f3529afe2e1e95988fac6fcf0b064761096018c3

File: com.github.ilai_deutel.kibi.yml (hunk 1 above)
Comment (ilai-deutel) [reply com.github.ilai_deutel.kibi.yml:11]: Correct, it is a terminal-only text editor

**Comment by ilai-deutel** [comment]
@barthalion Thank you, I added a `flathub.json` with `skip-appstream-check`

**Comment by barthalion** [comment]
bot, build com.github.ilai_deutel.kibi

**Pull request closed without merging**